buildAndWatch

```diff
@@ -12,6 +12,7 @@ function buildAndWatchWithNix() {
         exit 1
     fi
 
+    export LC_ALL=C.UTF-8       # fix locale error with Hakyll (see #29)
```

Comment:
Rather than exporting this for the entire script we could run the individual commands with this set. For example:
`nix-build -A builder && LC_ALL=C.UTF8 ./result/bin/…`

that would prevent the change from leaking outside of the command that needs it.